torch/testing/_internal/common_methods_invocations.py

```diff
@@ -9112,6 +9140,19 @@ def ref_pairwise_distance(input1, input2):
                DecorateInfo(unittest.skip("Skipped!"), 'TestJit', 'test_variant_consistency_jit'),
            ),
            supports_out=False,),
+    OpInfo('nn.functional.conv1d',
+           aliases=('conv1d',),
+           aten_name='conv1d',
+           dtypes=floating_types_and(torch.int64),
+           dtypesIfCUDA=floating_types_and(torch.float16, *[torch.bfloat16] if CUDA11OrLater else []),
+           sample_inputs_func=sample_inputs_conv1d,
+           gradcheck_nondet_tol=GRADCHECK_NONDET_TOL,
+           skips=(
+               # RuntimeError: !lhs.isAliasOf(rhs)INTERNAL ASSERT FAILED at
+               # "../torch/csrc/jit/passes/utils/check_alias_annotation.cpp":103, please report a bug to PyTorch.
+               DecorateInfo(unittest.skip("Skipped!"), 'TestJit', 'test_variant_consistency_jit'),
```

Comment:
Can this be an `expectedFailure`?

Reply:
Hi, @mruberry - Sorry for not getting back on this one. Having this as an `expectedFailure` is okay to me, will it be fine if I secretly push this in another OpInfo PR as a change? Apologies for the delay.

Reply:
Yes including little improvements with other PRs is always fine